### PR TITLE
Refactor lc_call_number_ender_on_index?

### PR DESCRIPTION
**Pull request by dkinzer**
Refactor to use present and only no returns.

### CODE REVIEW

**Comment by dkinzer** [comment]
No idea why this keeps falling the code should be equivalent

File: app/controllers/concerns/lc_classifications.rb [hunk 1]
```diff
@@ -4,11 +4,10 @@ module LCClassifications
...
-    return true unless params.dig("range", "lc_classification", "end").blank?
-    return true unless params.dig("f", "lc_outer_facet").blank?
-    return true unless params.dig("f", "lc_inner_facet").blank?
-    return true if params.dig("sort").present? && params["sort"].include?("lc_call_number_sort")
-    false
+    params.dig("range", "lc_classification", "begin").present? ||
+      params.dig("range", "lc_classification", "end").present? ||
+      params.dig("f", "lc_outer_facet").present? ||
+      params.dig("f", "lc_inner_facet").present? ||
+      !!(params.dig("sort").&include?("lc_call_number_sort"))
```
Comment (quoideneuf) [inline app/controllers/concerns/lc_classifications.rb:11]:
```suggestion
      !!(params.dig("sort")&.include?("lc_call_number_sort"))
```

Comment (dkinzer) [reply app/controllers/concerns/lc_classifications.rb:11]: 🤦

Comment (dkinzer) [reply app/controllers/concerns/lc_classifications.rb:11]: 🙇

**Review by quoideneuf — APPROVED** [review]

**Pull request merged**